src/mimemail.erl

```diff
@@ -81,9 +85,13 @@ decode(OrigHeaders, Body, Options) ->
 	Encoding = proplists:get_value(encoding, Options, none),
 	%FixedHeaders = fix_headers(Headers),
 	Headers = decode_headers(OrigHeaders, [], Encoding),
+	AllowMissingVersion = proplists:get_value(allow_missing_version, Options, false),
 	case parse_with_comments(get_header_value(<<"MIME-Version">>, Headers)) of
 		undefined ->
```

Comment:
Maybe move `AllowMissingVersion = proplists:get_value(allow_missing_version, Options, false),` to line after `undefined ->`? So, we can skip this proplists lookup when `MIME-Version` header is defined